collector/src/main.rs

```diff
@@ -995,113 +996,64 @@ fn main_result() -> anyhow::Result<i32> {
             bench_rustc,
             self_profile,
         } => {
-            println!("processing commits");
+            println!("processing artifacts");
             let client = reqwest::blocking::Client::new();
-            let response: collector::api::next_commit::Response = client
-                .get(&format!("{}/perf/next_commit", site_url))
+            let response: collector::api::next_artifact::Response = client
+                .get(&format!("{}/perf/next_artifact", site_url))
```

Comment:
Just want to double check - by the time this code runs, the API has been guaranteed to be updated as well, right?

Reply:
That's a question for @Mark-Simulacrum I suppose. But recently we have changed the API and it was fine. The bench runner is updated before running a benchmark, so that should be fine, I'm not sure how it's synchronized with also updating https://perf.rust-lang.org/ though.

Reply:
That's a question for @Mark-Simulacrum I suppose. But recently we have changed the API and it was fine. The bench runner is updated before running a benchmark, so that should be fine, I'm not sure how it's synchronized with also updating https://perf.rust-lang.org/ though.

Reply:
Yeah, there's not technically a guarantee here - but in practice it's only a few minutes difference (the collector will pull the git repo while the frontend waits for the docker build to finish in CI on master, and then gets a deployment on the new site). But in practice that just means a brief time when the collector won't work potentially, which is fine.